frontend/src/components/Menu/DropdownMenu.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { SimpleGrid } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import {
+  DropdownMenuSize,
+  DropdownMenuVariant,
+} from '../../theme/components/DropdownMenu'
+
+import {
+  DropdownMenu,
+  DropdownMenuButton,
+  DropdownMenuItem,
+  DropdownMenuList,
+} from './DropdownMenu'
+
+export default {
+  title: 'Components/DropdownMenu',
+  component: DropdownMenu,
+  argTypes: {
+    variant: {
+      options: ['outline', 'clear'],
+      control: { type: 'radio' },
+    },
+    size: {
+      options: ['normal', 'stretch'],
+      control: { type: 'radio' },
+    },
+  },
+} as Meta
+
+type DropdownMenuTemplateProps = {
+  variant: DropdownMenuVariant
+  size: DropdownMenuSize
+  children: string
+}
+
+type DropdownMenuGroupTemplateProps = {
+  variant: DropdownMenuVariant
+}
+
+const DropdownMenuTemplate: Story<DropdownMenuTemplateProps> = ({
```

Comment:
should we also have an interactive playground story with integration w/ react hook forms to demonstrate how the component is used? 

also, should we have a default `active` state (where the dropdown is open) to catch visual regressions easily?

Reply:
added, see if that works!